Classes/View/PreviewView.php

```diff
@@ -671,13 +671,32 @@ protected function parseGridColumnTemplate(array $row, Column $column, $colPosFl
 				$label = $column->getLabel();
 			}
 		}
+
+		// use this contents language as default
+		$columnLanguageUid = $row['sys_language_uid'];
+
+		// but for language mode all (uid -1):
+		if ( $columnLanguageUid == -1 ) {
```

Comment:
Please observe coding style requirements:

* Remove extra spaces inside the parenthesis of this "if"
* Remove indentation in variable assignments to put equals sign on same level
* Remove indentation in paranthesis of inline condition